pom.xml

```diff
@@ -88,7 +89,11 @@
                 <artifactId>subversion</artifactId>
                 <version>2.5</version>
             </dependency>
-
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>mercurial</artifactId>
+                <version>1.54</version>
```

Comment:
No, this is just `dependencyManagement`; the actual `dependency`s are `scope`d to `test`.